runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptionsTest.java

```diff
@@ -243,5 +243,12 @@ public void testDefaultGcpRegionFromGcloud() throws IOException, InterruptedExce
       DataflowPipelineOptions options = PipelineOptionsFactory.as(DataflowPipelineOptions.class);
       assertEquals("us-west1", options.getRegion());
     }
+
+    @Test
+    public void testHotKeyLoggingEnabled() {
+      DataflowPipelineOptions options = PipelineOptionsFactory.as(DataflowPipelineOptions.class);
+      options.setHotKeyLoggingEnabled(true);
+      assertEquals(true, options.getHotKeyLoggingEnabled());
+    }
```

Comment:
I don't believe there is value in testing this as PipelineOptionsFactoryTest covers these kinds of tests.
```suggestion
```